lib/modules/manager/mise/upgradeable-tooling.ts

```diff
@@ -174,3 +174,162 @@ export const miseTooling: Record<string, ToolingDefinition> = {
     },
   },
 };
+
+const miseRegistryTooling: Record<string, ToolingDefinition> = {
+  actionlint: {
+    misePluginUrl: 'https://mise.jdx.dev/registry.html#tools',
+    config: {
+      packageName: 'rhysd/actionlint',
+      datasource: GithubReleasesDatasource.id,
+      extractVersion: '^v(?<version>\\S+)',
+    },
+  },
+  'aws-cli': {
+    misePluginUrl: 'https://mise.jdx.dev/registry.html#tools',
+    config: {
+      datasource: GithubTagsDatasource.id,
+      packageName: 'aws/aws-cli',
+    },
+  },
+  'aws-vault': {
+    misePluginUrl: 'https://mise.jdx.dev/registry.html#tools',
+    config: {
+      datasource: GithubReleasesDatasource.id,
+      packageName: '99designs/aws-vault',
+      extractVersion: '^v(?<version>\\S+)',
+    },
+  },
+  buf: {
+    misePluginUrl: 'https://mise.jdx.dev/registry.html#tools',
+    config: {
+      packageName: 'bufbuild/buf',
+      datasource: GithubReleasesDatasource.id,
+      extractVersion: '^v(?<version>\\S+)',
+    },
+  },
+  consul: {
+    misePluginUrl: 'https://mise.jdx.dev/registry.html#tools',
+    config: {
+      packageName: 'hashicorp/consul',
+      datasource: GithubReleasesDatasource.id,
+      extractVersion: '^v(?<version>\\S+)',
+    },
+  },
+  hivemind: {
+    misePluginUrl: 'https://mise.jdx.dev/registry.html#tools',
+    config: {
+      packageName: 'DarthSim/hivemind',
+      datasource: GithubReleasesDatasource.id,
+      extractVersion: '^v(?<version>\\S+)',
+    },
+  },
+  jq: {
+    misePluginUrl: 'https://mise.jdx.dev/registry.html#tools',
+    config: {
+      packageName: 'jqlang/jq',
+      datasource: GithubReleasesDatasource.id,
+      extractVersion: '^jq-v(?<version>\\S+)',
+    },
+  },
+  kafka: {
+    misePluginUrl: 'https://mise.jdx.dev/registry.html#tools',
+    config: (version) => {
+      const apacheMatches = /^apache-(?<version>\d\S+)/.exec(version)?.groups;
+      if (apacheMatches) {
+        return {
+          datasource: GithubTagsDatasource.id,
+          packageName: 'apache/kafka',
+          currentValue: apacheMatches.version,
+          extractVersion: '^(?<version>\\S+)',
```

Comment:
is this really needed? please check all tools if they really need the extract version

Reply:
SGTM, I did go through all of the tools listed but this must've slipped through.